docroot/themes/humsci/humsci_basic/docs/utility-classes.md

```diff
@@ -134,6 +134,13 @@ For best practice, you might just want to use one responsive class if you'd like
 | hb-card-image-10-of-12 | ✅|
 | hb-card-image-11-of-12 | ✅|
 | hb-card-image-12-of-12 | ✅|
+
+#### Vertical Timeline within Views
```

Comment:
This could be combined with the table of classes on line 107-109. I don't think the vertical timeline needs its own separate section from other view specific classes. I'm also wondering if this needs to be view specific, couldn't this class be used on a component page as well?

Reply:
This class is already being applied within the component itself when you select the collapsed dropdown from paragraphs options, the reason it works as a View class is that the variant doesn't work inside of Views. Using this class allows us to apply that variant as a utility, while it could be added in a component section it is not needed as the component does it itself.

I will move it up with the other lines though, that makes sense.